app/src/main/java/hr/foi/air/sportloc/view/ui/activity/CommentActivity.java

```diff
@@ -0,0 +1,70 @@
+package hr.foi.air.sportloc.view.ui.activity;
+
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+
+
+import hr.foi.air.sportloc.R;
+import hr.foi.air.sportloc.view.ui.fragment.CommentListFragment;
+import android.support.v4.app.Fragment;
+import hr.foi.air.sportloc.view.ui.fragment.WriteCommentFragment;
+
+public class CommentActivity extends AppCompatActivity implements CommentListFragment.CommentsListViewListener{
+    Boolean openWriteComment;
+    Boolean userVote;
+    String stringId;
+    int id;
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+
+        openWriteComment = getIntent().getBooleanExtra("key", false);
+        userVote=getIntent().getBooleanExtra("vote",false);
+        stringId=getIntent().getStringExtra("stringId");
```

Comment:
try to keep hardcoded strings outside of the class, we have a final class Constants and you can just put it there. It makes it easier to use as you don't have to pay attention to the string itself when you type it into several locations. Also please do not be afraid of auto formatting.